source/administration/product-receipe-template.rst

```diff
@@ -0,0 +1,62 @@
+Product Recipe template
+========================
+
+This template defined how a product recipe can be added for the depending
+people.
+
+Problem Statement
+-------------------
+
+Define the problem statement. The problem statement could be anything from adding
+an architecture setup or installations.
+
+The problem statement should be written from user perspective and from organisation
+perspective.
+
+The user perspective might include
+
+- The requirements of the user
+- The environment of the user
+- The dependecies the user is having
+- The user case user would run the installation in
+
+The organisation perspective might include
+
+- How much effort the organisations adds in implementing the solution
+- How many work hours are put on implementing the solution for the user
+- Listing team putting the solution for the user
+- Dependecies with ongoing installations
+
+Pre-installation/pre-configuration notes
+----------------------------------------
+
+Define the pre-installation information. Projects installation has pre-installation
+steps describing the things you should setup when installating a requirement.
+Breifly describe the pre-installation or pre-configuration notes. Things you
+might include
+
+- Environment information
+- Version information
+- Release information
+- Dependecy information
+- Rollback information
+- Line of Business information
+
+Instructions Setup and Configuring the Solution
```

Comment:
Would you be open to clarifying the heading?

Reply:
The actual instructions and how the solution can be implemented.

Reply:
There is a ticker for this, I picked these works from the ticket.